database/migrations/2018_08_02_225416_create_place_categories_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
This migration has been already added by another task

https://github.com/BinaryStudioAcademy/bsa-2018-hedonist/blob/development/database/migrations/2018_08_02_225416_create_place_categories_table.php